clang/lib/CIR/CodeGen/CIRGenDecl.cpp

```diff
@@ -831,7 +831,12 @@ void CIRGenFunction::buildDecl(const Decl &D) {
 
   case Decl::Typedef:     // typedef int X;
   case Decl::TypeAlias: { // using X = int; [C++0x]
-    assert(0 && "Not implemented");
+    QualType Ty = cast<TypedefNameDecl>(D).getUnderlyingType();
+    if (auto *DI = getDebugInfo())
+      llvm_unreachable("NYI");
```

Comment:
You can change the debug info check into a `assert(!UnimplementedFeature::generateDebugInfo());`, just so that CIR doesn't crash if `-g` is used, but we still keep the entry point to easen up debug codegen work in the future.

Reply:
Ok, got it. Fixed